src/reducers/snackbar.js

```diff
@@ -0,0 +1,28 @@
+import { SUMMON_SNACKBAR, DISMISS_SNACKBAR } from '../actions/snackbar';
+
+export const initialState = {
+  message: '',
+  open: false,
+};
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case SUMMON_SNACKBAR: {
```

Comment:
Wondering if there is a better word than summon since it wasn't obvious to me at first without checking a dictionary (I'm a non-native English speaker). How about just OPEN_SNACKBAR?

Maybe a follow-up: we may also want to consider if we want to support setting multiple snackbars simultaneously.

Reply:
Great point – a lot of this got swept up into the changes that dawehner mentioned, so the semantics of the messaging should be resolved too. I also made the component work with multiple issued messages!